### PR TITLE
[CHI-345] Fix long strings in expansion panel content

### PR DESCRIPTION
**Pull request by davidojedacl**
@mattnickles

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/230/ for more details.

File: src/chi/components/expansion-panel/expansion-panel.scss [hunk 1]
```diff
@@ -26,6 +26,7 @@ $transition-duration: 0.2s;
 
     & > .m-epanel__content {
       flex: 1;
+      word-break:break-word;
```
Comment (mattnickles) [inline src/chi/components/expansion-panel/expansion-panel.scss]: @davidojedacl Can you add a space after the colon for code consistency?

Comment (davidojedacl) [reply src/chi/components/expansion-panel/expansion-panel.scss]: done

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/234/ for more details.

**Pull request merged**